import "funcjump.proto";
import "markedsym.proto";
import "staticsym.proto";

message BinPatch {
	required string		old_bid		= 1;
	required string		new_bid		= 2;
	optional string		new_path	= 3;
	repeated FuncJump	func_jumps	= 4;
	repeated MarkedSym	manual_symbols	= 5;
	repeated StaticSym	static_symbols	= 6;
	repeated MarkedSym	global_symbols	= 7;
	required string 	new_arch_type 	= 8;
}
